pages/gameboard/index.js

```diff
@@ -16,23 +16,27 @@ export default function Gameboard() {
     <PageContainer>
       <div className={styles.topRow}>
         <h1>SINGLE JEOPARDY</h1>
-        <div className={styles.playerScores}>
-          <div>Player1: $$$</div>
-          <div>Player2: $$$</div>
-          <div>Player3: $$$</div>
+        <div className={styles.playerScoresBorder}>
+          <div className={styles.playerScores}>
+            <div>Player1: $$$</div>
+            <div>Player2: $$$</div>
+            <div>Player3: $$$</div>
+          </div>
         </div>
         <Link href="/" className={styles.menuButton}>
           &#9776;
         </Link>
       </div>
-      <div className={styles.categoryRow}>
-        {mockCategories.map((category) => (
-          <div key={category} className={styles.categoryContainer}>
-            {category}
-          </div>
-        ))}
+      <div className={styles.gameboardBorder}>
+        <div className={styles.categoryRow}>
+          {mockCategories.map((category) => (
+            <div key={category} className={styles.categoryContainer}>
+              {category}
+            </div>
+          ))}
+        </div>
+        <ValueBoard />
       </div>
-      <ValueBoard />
     </PageContainer>
   );
```

Comment:
Before merging, let's modularize the categories and player scores into their own components.